pilot/cmd/pilot-agent/main.go

```diff
@@ -361,6 +365,16 @@ func init() {
 	proxyCmd.PersistentFlags().BoolVar(&disableInternalTelemetry, "disableInternalTelemetry", false,
 		"Disable internal telemetry")
 
+	// Flags for Pilot agent to take over Kubernetes readiness and liveness check.
+	proxyCmd.PersistentFlags().StringVar(&appLiveness.Path, "appLivenessPath", "",
```

Comment:
`appLiveness.URL`  we do not need host and port and separate params.
same is true for readiness.

Reply:
done.